sdk/storage/storage-file/review/storage-file.api.ts

```diff
@@ -0,0 +1,787 @@
+}
+
+// @public
+class FileClient extends StorageClient {
+  constructor(url: string, credential?: Credential, options?: NewPipelineOptions);
+  abortCopyFromURL(copyId: string, options?: FileAbortCopyFromURLOptions): Promise<Models.FileAbortCopyResponse>;
+  clearRange(offset: number, contentLength: number, options?: FileClearRangeOptions): Promise<Models.FileUploadRangeResponse>;
+  create(size: number, options?: FileCreateOptions): Promise<Models.FileCreateResponse>;
+  delete(options?: FileDeleteOptions): Promise<Models.FileDeleteResponse>;
+  download(offset?: number, count?: number, options?: FileDownloadOptions): Promise<Models.FileDownloadResponse>;
+  downloadToBuffer(buffer: Buffer, offset: number, count?: number, options?: DownloadFromAzureFileOptions): Promise<void>;
+  downloadToFile(filePath: string, offset?: number, count?: number, options?: FileDownloadOptions): Promise<Models.FileDownloadResponse>;
+  // (undocumented)
+  readonly filePath: string;
+  forceCloseHandle(handleId: string, options?: FileForceCloseHandlesOptions): Promise<Models.FileForceCloseHandlesResponse>;
+  forceCloseHandlesSegment(marker?: string, options?: FileForceCloseHandlesOptions): Promise<Models.FileForceCloseHandlesResponse>;
+  getProperties(options?: FileGetPropertiesOptions): Promise<Models.FileGetPropertiesResponse>;
+  getRangeList(options?: FileGetRangeListOptions): Promise<FileGetRangeListResponse>;
+  listHandlesSegment(marker?: string, options?: FileListHandlesSegmentOptions): Promise<Models.FileListHandlesResponse>;
+  resize(length: number, options?: FileResizeOptions): Promise<Models.FileSetHTTPHeadersResponse>;
+  setHTTPHeaders(fileHTTPHeaders?: FileHTTPHeaders, options?: FileSetHTTPHeadersOptions): Promise<Models.FileSetHTTPHeadersResponse>;
+  setMetadata(metadata?: Metadata, options?: FileSetMetadataOptions): Promise<Models.FileSetMetadataResponse>;
+  setProperties(properties?: FileProperties): Promise<SetPropertiesResponse>;
+  // (undocumented)
+  readonly shareName: string;
+  startCopyFromURL(copySource: string, options?: FileStartCopyOptions): Promise<Models.FileStartCopyResponse>;
+  uploadBrowserData(browserData: Blob | ArrayBuffer | ArrayBufferView, options?: UploadToAzureFileOptions): Promise<void>;
+  uploadFile(filePath: string, options?: UploadToAzureFileOptions): Promise<void>;
+  uploadRange(body: HttpRequestBody, offset: number, contentLength: number, options?: FileUploadRangeOptions): Promise<Models.FileUploadRangeResponse>;
+  uploadRangeFromURL(sourceURL: string, sourceOffset: number, destOffset: number, count: number, options?: FileUploadRangeFromURLOptions): Promise<Models.FileUploadRangeFromURLResponse>;
+  uploadResetableStream(streamFactory: (offset: number, count?: number) => NodeJS.ReadableStream, size: number, options?: UploadToAzureFileOptions): Promise<void>;
+  UploadSeekableBlob(blobFactory: (offset: number, size: number) => Blob, size: number, options?: UploadToAzureFileOptions): Promise<void>;
```

Comment:
>UploadSeekableBlob [](start = 2, length = 18)

should use lowercase `R`

Reply:
Which one? or you meant lower case `upload`?